pkg/commands/container.go

```diff
@@ -436,6 +450,11 @@ func (c *Container) Resize(fd uintptr) error {
 		Width:  uint(size.Width),
 	}
 
+	if fool {
```

Comment:
good idea, but unfortunately it doesn't work on iterm2 for me :( I still have to press enter to show the prompt. hmm, this is tricky

Reply:
Does the resizing even work at all? Try to write some characters, then resize terminal and see if lines wrap.

Reply:
if you do this:
```
		if sig == syscall.SIGWINCH {
			err := c.Resize(fd, false)
			if err != nil {
				return err
			}
			moveCursorRight := "\033[1C"
			hijack.Conn.Write([]byte(moveCursorRight))
```

It works for me. Only issue is, we don't actually want to send a command that interacts with the underlying process. If we could find an ansi escape code that causes a re-render, but doesn't actually change anything, that would be the proper solution!

Reply:
(didn't see your comment above). Yeah resizing works fine for me